pkg/services/ngalert/notifier/alertmanager.go

```diff
@@ -33,9 +35,27 @@ import (
 )
 
 const (
-	workingDir = "alerting"
+	pollInterval = 1 * time.Minute
+	workingDir   = "alerting"
 	// How long should we keep silences and notification entries on-disk after they've served their purpose.
 	retentionNotificationsAndSilences = 5 * 24 * time.Hour
+	// To start, the alertmanager needs at least one route defined.
+	// TODO: we should move this to Grafana settings and define this as the default.
+	alertmanagerDefaultConfiguration = `
+{
+	"alertmanager_config": {
+		"route": {
+			"receiver": "grafana-default-email"
+		},
+		"receivers": [{
+			"name": "grafana-default-email",
+			"email_configs": [{
```

Comment:
I'm a bit confused at the moment. If the API only accepts JSON at the top level what's the reasoning for using and supporting YAML within Grafana? I know we have to keep using YAML for sending to the Lotex API but the moment this hits grafana-managed backends it should all be JSON.

Reply:
I guess that the above comment is in reply to [this](https://github.com/grafana/grafana/pull/32851#pullrequestreview-633498596) one
That's because when I developed persisting the configuration to the database you told me that we wanted it to save it in YAML because it's cleaner/stricter and also I assumed that was an alert manager requirement.
So I did, even though I didn't like it a lot but I'm happy now to see this go away :)

Reply:
Yay!